src/main/java/org/dita/dost/invoker/ArgumentParser.java

```diff
@@ -326,7 +327,9 @@ public Arguments processArgs(final String[] arguments) {
                 targets.addElement(arg);
             }
         }
-        if (!inputs.isEmpty()) {
+        if (inputs.size() > 1) {
+            definedProps.put("args.inputs", String.join(" ", inputs));
```

Comment:
Is this going to cause issues with inputs that have spaces in the file name or path? Or do those get normalized to `%20` before we reach this point?

Reply:
It's up to the client to pass valid URIs. So if the end user user inputs an invalid URI (a URI with a space) to the client, the client will need to convert it to a valid URI (by escaping the space with a `%20`) when setting `args.inputs` property.